app/views/api/v2/template_inputs/main.json.rabl

```diff
@@ -2,7 +2,8 @@ object @template_input
 
 extends "api/v2/template_inputs/base"
 
-attributes :template_id, :fact_name, :variable_name, :puppet_parameter_name
+attributes :template_id, :fact_name, :variable_name, :puppet_parameter_name, :puppet_class_name
+attributes :description, :required
```

Comment:
It can span multiple lines here with only 1 attributes call

```ruby
attributes :template_id, :fact_name, :variable_name, :puppet_parameter_name,
  :puppet_class_name, :description, :require
```